pom.xml

```diff
@@ -0,0 +1,40 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
The repository is assumed to contains lot of components, projects with different technologies. If you want to have a specific project / module with specific technologies like java, please create directories for itself.